hibikase/PlaybackWidget.cpp

```diff
@@ -94,7 +117,8 @@ void PlaybackWidget::OnStateChanged(QAudio::State state)
 void PlaybackWidget::UpdateTime(std::chrono::milliseconds ms)
 {
     QString text;
-    if (ms.count() >= 0)
+    // If worker has been deleted, ignore time update
```

Comment:
Could you explain in what situation UpdateTime would be called when m_worker has been deleted?

Reply:
It shouldn't be, but it happens for some reason. I guess queued events.

Reply:
There is the call in OnStateChanged, I suppose, but that one should always have a count of -1 anyway...

Reply:
I suppose it's fine, then. I'm just thinking about if it's weird to have a comment like this that applies to the first half of the condition but not the second half...